numba_cuda/numba/cuda/compiler.py

```diff
@@ -307,8 +307,11 @@ def compile(pyfunc, sig, debug=False, lineinfo=False, device=True,
     if output not in ("ptx", "ltoir"):
         raise NotImplementedError(f'Unsupported output type: {output}')
 
+    debug = config.CUDA_DEBUGINFO_DEFAULT if debug is None else debug
+    opt = (config.OPT._raw_value != 0) if opt is None else opt
```

Comment:
I think the opt level is an int subclass, so we can avoid using a private property:
```suggestion
    opt = (config.OPT != 0) if opt is None else opt
```

Reply:
as it is.